Classes/BITCrashManager.m

```diff
@@ -198,9 +198,8 @@ - (instancetype)init {
     
     _crashManagerStatus = BITCrashManagerStatusAlwaysAsk;
     
-    NSString *testValue = [[NSUserDefaults standardUserDefaults] stringForKey:kBITCrashManagerStatus];
-    if (testValue) {
-      _crashManagerStatus = (BITCrashManagerStatus) [[NSUserDefaults standardUserDefaults] integerForKey:kBITCrashManagerStatus];
+    if ([[NSUserDefaults standardUserDefaults] stringForKey:kBITCrashManagerStatus]) {
+      _crashManagerStatus = (BITCrashManagerStatus)[[NSUserDefaults standardUserDefaults] integerForKey:kBITCrashManagerStatus];
```

Comment:
@chrwend I switched this back from using `valueForKey:` to `stringForKey:`. `valueForKey:` is a Key-Value Coding method that could have other side effects.
The actual value is an integer but we can't use `integerForKey:` in the `if (…)` condition because it can't return `nil`, only `0` which might be an actually valid value. Therefore we have to use two different methods for checking if the value exists and then actually getting it.